compiler/resolution/functionResolution.cpp

```diff
@@ -4720,7 +4720,6 @@ static bool
 isNormalField(Symbol* field)
```

Comment:
That is an excellent suggestion. I also considered renaming the function but didn't have an obvious good name and wanted to keep the patch shorter... I'll add a comment.